libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/internal/kapt/Kapt3KotlinGradleSubplugin.kt

```diff
@@ -74,11 +74,14 @@ class Kapt3GradleSubplugin @Inject internal constructor(private val registry: To
         fun getKaptGeneratedKotlinSourcesDir(project: Project, sourceSetName: String) =
             File(project.buildDir, "generated/source/kaptKotlin/$sourceSetName")
 
-        private val VERBOSE_OPTION_NAME = "kapt.verbose"
-        private val USE_WORKER_API = "kapt.use.worker.api"
-        private val INFO_AS_WARNINGS = "kapt.info.as.warnings"
-        private val INCLUDE_COMPILE_CLASSPATH = "kapt.include.compile.classpath"
-        private val INCREMENTAL_APT = "kapt.incremental.apt"
+        // Pairs of property names and their default values.
+        // The default values of these properties should typically match those defined in org.jetbrains.kotlin.base.kapt3.KaptFlag,
+        private val VERBOSE_OPTION_NAME = Pair("kapt.verbose", false)
+        private val USE_WORKER_API = Pair("kapt.use.worker.api", true)
+        private val INCREMENTAL_APT = Pair("kapt.incremental.apt", true) // Currently doesn't match KaptFlag, not sure if it's intended.
```

Comment:
Inconsistency detected: This flag is disabled by default according to the code at `org.jetbrains.kotlin.base.kapt3.KaptOptions.Builder#getFlags`--see below. // cc @gavra0

```
        // This set includes enabled-by-default flags. It doesn't contain INCREMENTAL_APT.
        val flags: MutableSet<KaptFlag> = mutableSetOf(
            KaptFlag.USE_LIGHT_ANALYSIS,
            KaptFlag.INCLUDE_COMPILE_CLASSPATH
        )
```

Reply:
Cool, I've added a commit to add a comment for that: https://github.com/JetBrains/kotlin/pull/3942/commits/571b63ac10fa73c8c33c622928dc77c76d48abd5

(@nav-nav Nataliya told me she prefers avoiding force pushes, and will help with squashing commits.)